src/components/MyAccount/index.tsx

```diff
@@ -369,6 +373,24 @@ const MyAccountContent = ({
                 </Grid>
               </>
             )}
+            {weightUnitsEnabled && (
+              <Grid item xs={isMobile ? 12 : 4} sx={{ '&.MuiGrid-item': { paddingTop: theme.spacing(3) } }}>
```

Comment:
i think you want margin top 3 in mobile only.. in desktop view it shares same row as time zones and the margin top is not consistent

Reply:
Good catch!